phpstan.neon.dist

```diff
@@ -75,7 +66,7 @@ parameters:
                 - lib/Doctrine/DBAL/Driver/Mysqli/MysqliStatement.php
 
         -
-            message: '~Method Doctrine\\DBAL\\Driver\\PDOSqlsrv\\Connection\:\:lastInsertId\(\) should return string but returns int\|string\|false\|null\.~'
+            message: '~Method Doctrine\\DBAL\\Driver\\PDOSqlsrv\\Connection\:\:lastInsertId\(\) should return string\|false but returns int\|string\|false\|null\.~'
```

Comment:
If the underlying implementation can return an `int`, then the DBAL API can be documented as returning an `int` as well. It doesn't look like a false-positive issue to be suppressd.

Reply:
The false positive here comes from the fact that the return value of `fetchColumn()` is returned directly.

Reply:
I see. It's still a false-positive but with a slightly different error message, hence the change.